js/compiler.ts

```diff
@@ -541,6 +637,50 @@ window.compilerMain = function compilerMain() {
   };
 };
 
+const decoder = new TextDecoder();
+
+function getCompilerConfig(
+  compilerType: string
+): { path: string; data: string } {
+  const builder = flatbuffers.createBuilder();
+  const compilerType_ = builder.createString(compilerType);
+  msg.CompilerConfig.startCompilerConfig(builder);
+  msg.CompilerConfig.addCompilerType(builder, compilerType_);
+  const inner = msg.CompilerConfig.endCompilerConfig(builder);
+  const baseRes = sendSync(builder, msg.Any.CompilerConfig, inner);
+  assert(baseRes != null);
+  assert(msg.Any.CompilerConfigRes === baseRes!.innerType());
+  const res = new msg.CompilerConfigRes();
+  assert(baseRes!.inner(res) != null);
+
+  const path = res.path()!;
+  assert(path != null);
+  const dataArray = res.dataArray()!;
+  assert(dataArray != null);
+  const data = decoder.decode(dataArray);
+  return { path, data };
+}
+
 export default function denoMain(): void {
   os.start("TS");
+  const { path, data } = getCompilerConfig("typescript");
+  if (data.length) {
+    log(`Configuring compiler from "${path}"`);
+    const ignoredOptions = compiler.configure(path, data);
+    if (ignoredOptions) {
+      if (os.noColor) {
+        console.warn(
+          `Unsupported compiler options in "${path}"\n` +
+            `  The following options were ignored:\n` +
+            `    ${ignoredOptions.join(", ")}`
```

Comment:
Thank you for doing this!
Can you show in warning which configuration parameters are used by default?
this would be very convenient!

Reply:
They would be very chatty.  It can be logged out in debug.  I wasn't even sure about this as it also would be chatty a lot of the time.